plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJdbcConnectorTest.java

```diff
@@ -256,6 +256,16 @@ public void testTableWithOnlyUnsupportedColumns()
         }
     }
 
+    @Override
+    public void testNativeQueryColumnAlias()
+    {
+        // override because H2 uppercase column names by default
+        assertThat(query(format("SELECT * FROM TABLE(system.query(query => 'SELECT name AS region_name FROM %s.region WHERE regionkey = 0'))", getSession().getSchema().orElseThrow())))
+                .skippingTypesCheck()
```

Comment:
Why skip types check?

Reply:
Type check is unrelated to this test. Otherwise, the test will fail due to length difference: `varchar(25)` vs `varchar(6)`

Reply:
You need to use CAST(… as varchar(25)) in the .matches() clause. The assertion framework performs strict type checking.

Reply:
I knew and didn't add the cast intentionally. I will fix it though.